enterprise_access/apps/api/v1/views/subsidy_access_policy.py

```diff
@@ -566,22 +591,41 @@ def can_redeem(self, request, enterprise_customer_uuid):
                         "reason": reason,
                         "policy_uuids": [policy.uuid for policy in policies],
                     })
+
+            # Select one redeemable policy (i.e. use the "policy resolver") and serialize it for output.
             if redeemable_policies:
                 resolved_policy = SubsidyAccessPolicy.resolve_policy(redeemable_policies)
-                serialized_policy = serializers.SubsidyAccessPolicyRedeemableResponseSerializer(resolved_policy).data
 
+            # Determine if the learner has already redeemed the requested content_key.
             has_successful_redemption = any(
                 redemption['state'] == TransactionStateChoices.COMMITTED
                 for redemption in redemptions
             )
-            can_redeem_for_content_response = {
+
+            # Determine the price for content for display purposes only.
+            try:
+                content_metadata = self.subsidy_client.get_subsidy_content_data(enterprise_customer_uuid, content_key)
+                # Note that the "content_price" key is guaranteed to exist, but the value may be None.  Also, the
+                # upstream field type is a string representing a float, and we passthrough that verbatim.
+                content_price = content_metadata["content_price"]
```

Comment:
FYI, I just looked at the API spec and discovered that integer cents was called for, not a passthrough string representing decimal dollars. https://github.com/openedx/enterprise-access/blob/6ad37aac9b3fbdf594c2409fd8b7ad8aa7ab3555/docs/decisions/0006-api-specification-for-enterprise-mfes.rst#L107

I plan to convert this before merging.